### PR TITLE
ApiService: Export endpoint

### PR DESCRIPTION
**Pull request by zannkukai**
Adds method to retrieve an export endpoint API based on resource type.

Co-authored-by: Renaud Michotte <renaud.michotte@gmail.com>

### CODE REVIEW

**Review by lauren-d — APPROVED** [review]

File: projects/rero/ng-core/src/lib/api/api.service.ts [hunk 1]
```diff
@@ -45,19 +43,32 @@ export class ApiService {
...
 
+  /**
+   * Returns Invenio-RERO-base API export endpoint corresponding to type.
+   *
+   * @param type Type of the resource.
+   * @param absolute If absolute or relative url must be returned.
+   * @return Endpoint as string.
+   */
+  getExportEndpointByType(type: string, absolute: boolean = false): string {
+    let endpoint = this.endpointPrefix + '/export/' + type;
```
Comment (Garfield-fr) [inline projects/rero/ng-core/src/lib/api/api.service.ts:68]: there should be a configuration to allow customization of this entry

**Review by Garfield-fr — COMMENTED** [review]
This is specific to `rero-ils`. Suggestion: put this in `rero-ils-ui`.

File: projects/rero/ng-core/src/lib/api/api.service.ts (hunk 1 above)
Comment (zannkukai) [reply projects/rero/ng-core/src/lib/api/api.service.ts:68]: This implies that this export endpoint URL should be customizable but at this time, it isn't. The new export module from 'invenio-rero-base' creates endpoint based on resource "list_route" prepend with static "/export" string.

Comment (jma) [reply projects/rero/ng-core/src/lib/api/api.service.ts:68]: You an use back quotes instead of string concatenation.

**Review by jma — APPROVED** [review]

**Pull request merged**